release-notes.md

```diff
@@ -0,0 +1,60 @@
+# Release v0.9.0
+
+## New Features
+* Add pai service dashboard to grafana. - [PR 1694](https://github.com/Microsoft/pai/pull/1694)
+* Webportal plugin implementation and change pai marketplace to plugin. -[PR 1700](https://github.com/Microsoft/pai/pull/1700)
+* Support update virtual cluster dynamically from webportal. -[PR 1831](https://github.com/Microsoft/pai/pull/1831) -[PR 1974](https://github.com/Microsoft/pai/pull/1974)
+
+
+## Improvements
+### Service
+* Cluster object model implementation to make it easier for developer to add customized service configuration generation logic. -[PR 1735](https://github.com/Microsoft/pai/pull/1735)
+* Job exporter refactor to avoid single external command call will make exporter hang indefinitely. -[PR 1840](https://github.com/Microsoft/pai/pull/1840)
+* Make drivers version configurable. - [PR 1626](https://github.com/Microsoft/pai/pull/1626)
+* Extend yarn local log expiration time to 7 days. -[PR 1673](https://github.com/Microsoft/pai/pull/1673)
+* Add two driver images. Current supports driver's versions are 384.111, 390.25 and 410.73. By default will deploy 390.25 version. -[PR 1642](https://github.com/Microsoft/pai/pull/1642)
+* Reduce grafana image from 440M to 280M by merging all startup scripts add removing useless plugin. -[PR 1685](https://github.com/Microsoft/pai/pull/1685)
+* Upgrade Nodejs version of webportal and rest server to 8. -[PR 1453](https://github.com/Microsoft/pai/pull/1453)
+
+### Job
+* Add timestamp for cloned job's name - [PR 1532](https://github.com/Microsoft/pai/pull/1532)
+* Add log if job's image doesn't have ssh server - [PR 1675](https://github.com/Microsoft/pai/pull/1675)
+
```

Comment:
https://github.com/Microsoft/pai/pull/1860

Escape injected variables in shell scripts